util/fqdn.go

```diff
@@ -2,13 +2,18 @@ package util
 
 import (
 	"net"
-	"regexp"
 	"strings"
+	"github.com/asaskevich/govalidator"
 )
 
 func IsFQDN(domain string) bool {
-	RegExp := regexp.MustCompile(`^((([a-zA-Z]{1})|([a-zA-Z]{1}[a-zA-Z]{1})|([a-zA-Z]{1}[0-9]{1})|([0-9]{1}[a-zA-Z]{1})|([a-zA-Z0-9][a-zA-Z0-9-_]{1,61}[a-zA-Z0-9])|\*)\.){2,}([a-zA-Z]{2,6}|[a-zA-Z0-9-]{2,30}\.[a-zA-Z]{2,3})(\.)*$`)
-	return RegExp.MatchString(domain)
+	var ret bool
```

Comment:
I don't think you can have that, or it's at the least undefined behavior. I can't seem to find it in certificates either. We can add a check for it but it looks like it wasn't a big deal in this PR. https://github.com/zmap/zgrab/pull/188. thoughts?